src/overlays/Drawer.scss

```diff
@@ -0,0 +1,73 @@
+.seeds-drawer {
+  --drawer-border-color: var(--seeds-border-color);
+  --drawer-padding: var(--seeds-s6);
+  --drawer-close-icon-color: var(--seeds-color-primary);
+  --drawer-header-background-color: var(--seeds-bg-color-surface);
+  --drawer-header-padding-block: calc(var(--drawer-padding) - var(--seeds-s2));
+  --drawer-header-padding-inline: var(--drawer-padding);
+  --drawer-content-text-color: var(--seeds-text-color);
+  --drawer-content-background-color: var(--seeds-bg-color-canvas);
+  --drawer-footer-background-color: var(--drawer-content-background-color);
+  --drawer-footer-padding-block: var(--drawer-padding);
+  --drawer-footer-padding-inline: var(--drawer-padding);
+
+  --overlay-border-color: var(--drawer-border-color);
+  --overlay-padding: var(--drawer-padding);
+  --overlay-close-icon-color: var(--drawer-close-icon-color);
+  --overlay-header-background-color: var(--drawer-header-background-color);
+  --overlay-header-padding-block: var(--drawer-header-padding-block);
+  --overlay-header-padding-inline: var(--drawer-header-padding-inline);
+  --overlay-content-text-color: var(--drawer-content-text-color);
+  --overlay-content-background-color: var(--drawer-content-background-color);
+  --overlay-footer-background-color: var(--drawer-footer-background-color);
+  --overlay-footer-padding-block: var(--drawer-footer-padding-block);
+  --overlay-footer-padding-inline: var(--drawer-footer-padding-inline);
+
+  --drawer-width-sm: 100vw;
+  --drawer-width-md: 48rem;
+  --drawer-width-lg: 64rem;
+  --drawer-width-xl: 80rem;
+  --drawer-nested-width-sm: 95vw;
+  --drawer-nested-width-md: 38rem;
+  --drawer-nested-width-lg: 54rem;
+  --drawer-nested-width-xl: 70rem;
+
+  inset-inline-start: auto;
+  height: 100vh;
+
+  @media (--sm-only) {
+    width: var(--drawer-width-sm);
+  }
+  @media (--md-and-up) {
+    width: var(--drawer-width-md);
+  }
+  @media (--lg-and-up) {
+    width: var(--drawer-width-lg);
+  }
+  @media (--xl-and-up) {
+    width: var(--drawer-width-xl);
+  }
```

Comment:
There is a bug when you are between 676px and 640px the drawer goes left of the page. https://deploy-preview-59--storybook-ui-seeds.netlify.app/iframe.html?args=&id=overlays-drawer--many-buttons&viewMode=story